ovirt-guest-agent-fedora/ovirt-container-hibernate.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/usr/bin/nsenter --mount=/proc/1/ns/mnt -- /usr/share/ovirt-guest-agent/hibernate $@
```

Comment:
also, why do we need to go into the mount namespace of PID1?

If `hibernate` needs to write to /sys/power/state, we can probably mount `/sys` as rw instead of ro?